pkg/operator/ceph/cluster/cluster.go

```diff
@@ -32,7 +32,7 @@ import (
 
 	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
 	"github.com/rook/rook/pkg/clusterd"
-	"github.com/rook/rook/pkg/daemon/ceph/client"
+	cephclient "github.com/rook/rook/pkg/daemon/ceph/client"
```

Comment:
Why is this name change needed here? I don't see any conflicts that might need this change.

Reply:
right, reverted